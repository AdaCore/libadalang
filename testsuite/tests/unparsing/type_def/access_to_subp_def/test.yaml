driver: unparser
rule: subp_decl
description: |
  Test that `f_has_not_null`, `f_has_protected` and `f_subp_spec` do not have
  line breaks between them. Also test that `f_subp_spec` has its parameters
  correctly indented.
