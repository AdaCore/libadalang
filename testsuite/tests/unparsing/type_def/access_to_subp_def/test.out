procedure Find_Matching_Parents
  (Match_1 :
     access protected function
       (Node : Libadalang.Analysis.Ada_Node'Class) return Boolean;
   Match_2 :
     not null access protected function
       (Node : Libadalang.Analysis.Ada_Node'Class) return Boolean);
